packages/carousel/src/Default/stories/Default-index.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { component as Carousel } from '../index';
+import { CarouselDefaultProps, Slide } from '../Default.types';
+
+import { appearance } from './Default-appearance.stories';
+import { size } from './Default-size.stories';
+import { theme } from './Default-theme.stories';
+
+const slides: Slide[] = [
+  {
+    id: '1',
+    style: { backgroundColor: '#017aba' },
+    backgroundContent: '',
+    caption: (
+      <>
+        <h3>Slide 1 label</h3>
+        <p>Slide 1 caption text. Text example.</p>
+      </>
+    ),
+    bsItemProps: {
+      interval: 3000,
+    },
+  },
+  {
+    id: '2',
+    style: { backgroundColor: '#6a98b0' },
+    backgroundContent: '',
+    caption: (
+      <>
+        <h3>Slide 2 label</h3>
+        <p>Slide 2 caption text. Text example.</p>
+      </>
+    ),
```

Comment:
Is this structure just one option of what we could render on an individual slide? Just wondering why we would return this fragment in the slide props instead of adding the h2 and p tags as part of the main structure of the component

Reply:
That's just an option of what could be rendered, we could have multiple slides in the carousel with different content, like:

Slide 1:
```
      <>
        <h3>Slide 1 label</h3>
        <p>Slide 1 caption text. Text example.</p>
      </>
```
Slide 2:
```
      <>
        <Icon icon='gear' />
        <p>Slide 2 caption text. Text example.</p>
        <Button>Click me!</Button>
      </>
```